README.md

```diff
@@ -277,13 +277,16 @@ Show channel name in hotlist after activity
 ```
 /set weechat.look.hotlist_names_level 14
 ```
+Development
+-----------------
 
-Enable debug mode and change debug level (default 3, decrease to increase logging and vice versa):
+Enable debug mode and look in `/tmp/weeslack-debug/` for the json dumps from the websockets:
 ```
-/set plugins.var.python.slack.debug_mode on
-/set plugins.var.python.slack.debug_level 2
```

Comment:
Please don't remove this. `record_events` only record the responses of the requests, while the debug mode logs more about what wee-slack is doing.

Reply:
oh, I was getting errors on the debug. I'll back out the README changes into a new PR. I pushed them in while high on cold and flu tablets.